api/schemas/flow-video.json

```diff
@@ -40,7 +40,8 @@
                             "properties": {
                                 "numerator": {
                                     "description": "numerator",
-                                    "type": "integer"
+                                    "type": "integer",
+                                    "exclusiveMinimum": 0
```

Comment:
Might be useful to add a clarification to the description that if `frame_rate` is not provided then either it is unknown or it is known to be variable frame rate. Not sure, but we could think about whether a zero frame rate is used to indicate a variable frame rate rather than lump it in with unknown?

Reply:
I wonder if we would be better signalling it explicitly through a separate parameter? Either way, we should probably wait until we have more concrete use cases before making that decision.